azure-pipelines.yml

```diff
@@ -261,20 +261,22 @@ jobs:
 
 
   - bash: |
-       echo == Remove pre-installed azure-cli ==
-       brew uninstall azure-cli
+      set -ev
```

Comment:
Should we verify that all `-bash:` entries in the YAML have this annotation?  If the logic were cut/paste from the old GUI-based components and the GUI component behaved differently, then this could affect many other checks.

Reply:
Yes, absolutely. I'm planning on submitting another PR with those changes as soon as this is resolved.

Reply:
Looks like all other instances had at least `set -e` with some combination of  `xv` as well.